controllers/reconcilers/configuration/grafana.go

```diff
@@ -18,6 +18,11 @@ func (r *Reconciler) reconcileGrafanaCr(ctx context.Context, cr *v1.Observabilit
 	var f = false
 	var t = true
 
+	version := "docker.io/grafana/grafana:" + model.GetGrafanaVersion(indexes)
+	if version == "docker.io/grafana/grafana:" {
+		version = ""
+	}
```

Comment:
I had the same idea initially but if a BaseImage: tag is defined and not empty like there is anycharacter in there it will try to pull down a docker image that does not exist. But setting it to being empty will ignore the BaseImage tag and not add it to the yaml then resulting in the hardcoded grafana image being used.